Document-Processing/PDF/PDF-Library/NET/Working-with-Pages.md

```diff
@@ -442,6 +442,104 @@ document.Close(True)
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/Pages/Adding-sections-with-different-page-settings/). 
 
+## Set section page numbers with custom styles in PDFs
```

Comment:
Customize section page numbering styles in PDF documents

Reply:
Changed